#!/usr/bin/env python
PACKAGE = "artificial_potential_field_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("attraction_gain", double_t, 0, "A double parameter",    50, 0,   100)
gen.add("repulsion_gain", double_t, 0, "A double parameter",    500, 0,   1000)
gen.add("radius", double_t, 0, "A double parameter",    10, 0,   30)
gen.add("forcemap_width", double_t, 0, "A double parameter",    10, 0,   30)
gen.add("forcemap_height", double_t, 0, "A double parameter",    10, 0,   30)

exit(gen.generate(PACKAGE, "artificial_potential_field_planner", "APFPlanner"))
